### PR TITLE
i18n(zh-tw): Add `editor-setup.md` and update other translations

### PR DESCRIPTION
**Pull request by asgoshawk**
#### What kind of changes does this PR include?

- Minor content fixes (broken links, typos, etc.)
- New or updated content
- Translated content

#### Description

- Add `src/pages/zh-tw/editor-setup.md` translation.
- Update `src/i18n/zh-tw/nav.ts` to make the words consistent.
- Update `src/i18n/zh-tw/ui.ts` with the latest dictionary.
- Update `src/pages/zh-tw/getting-started.md` with package manager tabs and add `i18nReady: true`.

### CODE REVIEW

File: src/pages/zh-tw/editor-setup.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+讓我們直接開始，今天就來安裝 [Astro VS Code 延伸模組](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode) 吧。
+
+📚 看看如何在你的 Astro 專案[設定 TypeScript](/zh-tw/guides/typescript/)。
+
+## 其他程式編輯器
+
+我們厲害的社群也為了其他有名的編輯器維護了數個延伸模組，以下包括：
+
+- [Open VSX 的 VS Code 延伸模組](https://open-vsx.org/extension/astro-build/astro-vscode) <span style="margin: 0.25em;"><Badge variant="accent">Official</Badge></span> - 官方的 Astro VS Code 延伸模組，對部分開放平台如 [VSCodium](https://vscodium.com/) 可到 Open VSX regitry 取得
```
Comment (delucis) [inline src/pages/zh-tw/editor-setup.md:30]: Looks like a small typo here

```suggestion
- [Open VSX 的 VS Code 延伸模組](https://open-vsx.org/extension/astro-build/astro-vscode) <span style="margin: 0.25em;"><Badge variant="accent">Official</Badge></span> - 官方的 Astro VS Code 延伸模組，對部分開放平台如 [VSCodium](https://vscodium.com/) 可到 Open VSX registry 取得
```

**Review by delucis — COMMENTED** [review]
Thanks @asgoshawk! Spotted one small typo:

File: src/pages/zh-tw/editor-setup.md (hunk 1 above)
Comment (asgoshawk) [reply src/pages/zh-tw/editor-setup.md]: I've update for that. Thanks a lot!

**Review by delucis — APPROVED** [review]
Thanks @asgoshawk! 🚀

**Pull request merged**